new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioCrossDomainTransaction.java

```diff
@@ -0,0 +1,426 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createImageAndVerify;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.dockerLoginAndPushImageToRegistry;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.generateFileFromTemplate;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.ExecCommand.exec;
+import static oracle.weblogic.kubernetes.utils.IstioUtils.deployHttpIstioGatewayAndVirtualservice;
+import static oracle.weblogic.kubernetes.utils.IstioUtils.getIstioHttpIngressPort;
+import static oracle.weblogic.kubernetes.utils.IstioUtils.installIstio;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Verify cross domain transaction with istio enabled is successful")
+@IntegrationTest
+public class ItIstioCrossDomainTransaction implements LoggedTest {
+
+  private static final String WDT_MODEL_FILE_DOMAIN1 = "model-crossdomaintransaction-domain1.yaml";
+  private static final String WDT_MODEL_FILE_DOMAIN2 = "model-crossdomaintransaction-domain2.yaml";
+
+  private static final String WDT_MODEL_DOMAIN1_PROPS = "model-crossdomaintransaction-domain1.properties";
+  private static final String WDT_MODEL_DOMAIN2_PROPS = "model-crossdomaintransaction-domain2.properties";
+  private static final String WDT_IMAGE_NAME1 = "domain1-wdt-image";
+  private static final String WDT_IMAGE_NAME2 = "domain2-wdt-image";
+  private static final String WDT_APP_NAME = "txforward";
+  private static final String PROPS_TEMP_DIR = RESULTS_ROOT + "/istiocrossdomaintransactiontemp";
+
+  private static HelmParams opHelmParams = null;
+  private static String opNamespace = null;
+  private static String operatorImage = null;
+  private static String domain1Namespace = null;
+  private static String domain2Namespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+  private static String dockerConfigJson = "";
+  private String domainUid1 = "domain1";
+  private String domainUid2 = "domain2";
+  private static Map<String, Object> secretNameMap;
+  private final String domain1AdminServerPodName = domainUid1 + "-admin-server";
+  private final String domain1ManagedServerPrefix = domainUid1 + "-managed-server";
+  private final String domain2ManagedServerPrefix = domainUid2 + "-managed-server";
+  private String clusterName = "cluster-1";
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *     JUnit engine parameter resolution mechanism
```

Comment:
now a simple description is good enough I guess
```suggestion
   @param namespaces injected by JUnit
```

Reply:
done